graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH8.kt

```diff
@@ -0,0 +1,90 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctionalElements
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+
+@BEH(GraphBehaviours.CreateDirectedGraph, GraphFunctionalElements.DirectedGraph, """
+    Create a directed graph
+""")
+class BEH8 : GraphScenario() {
+
+    @Eac("A directed graph is a graph that only contains directional edges",
+        """When a directional edge is declared a path can only be walked in one order
+        """)
+    fun directionalEdge() {
+        given {
+            thereIsADirectedGraph(graphA) {
+                edge('a', 'b')
+            }
+        }
+        then {
+            pathExists(graphA, from = 'a', to = 'b')
+            noPathExists(graphA, from = 'b', to = 'a')
+        }
+    }
+
+    @Eac("A graph may any positive number of vertices")
+    fun withVertices() {
+        given {
+            thereIsADirectedGraph(graphA) {
+                vertex("a")
+                vertex("b")
+                vertex("c")
+            }
+        }
+        then {
+            hasVertexCount(graphA, 3)
+        }
+    }
+
+    @Eac("A graph may contain no vertices")
```

Comment:
⚠️ Again, I'm not convinced this is an EAC as it stands. By analogy with the tic-tac-toe example, I would say something like
> A newly created graph is empty and has neither vertices nor edges
https://github.com/anaplan-engineering/azuki/blob/e8e48aca470cf46f4a41cd5f4db7166261edd8c6/tictactoe/eacs/src/test/kotlin/com/anaplan/engineering/azuki/tictactoe/eacs/BEH1.kt#L15-L24